designer/server/src/main/scala/pl/touk/nussknacker/ui/api/TestingApiErrorMessages.scala

```diff
@@ -0,0 +1,41 @@
+package pl.touk.nussknacker.ui.api
+
+object TestingApiErrorMessages {
+
+  val noSourcesWithTestDataGeneration = "No sources with test data generation available"
+
+  val noDataGenerated = "No data was generated"
```

Comment:
This message is funny (I know that it is not yours). Take a look:
- We ask test generator to generate data
- Test generators answers: no data was generated

The obvious question is, why? your are a machine, generate it for me! :)

The root problem is that we don't really generate data, but we provide some data samples. In all situations sample of live data, but it is another story.
To make it clear, we can:
1. Start to call this operation "fetch sample of live data" instead of generate data. Then the message would be "No sample live data found"
2. Call this operation "provide sample data" without specifying that it is fetched and it is live. In this case the message would be "Cannot provide sample data. Possible cause: no live sample data available"

Reply:
Changed to second suggestion.